src/app.rs

```diff
@@ -157,6 +157,18 @@ impl ApplicationConfig {
     }
 }
 
+const INTERNAL_CONFIG: &str = r#"
+sources:
+  _internal_metrics:
+    type: internal_metrics
+sinks:
+  _internal_console:
+    type: console
+    inputs: [_internal_metrics]
+    encoding:
+      codec: text
+"#;
```

Comment:
This is nice and succinct, but I do wonder if instantiating the actual config types and config builder would be a little cleaner.

Reply:
Done in f00d09c44. It is quite a bit more verbose, too, with types pulled in from all over the place.